entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java

```diff
@@ -167,6 +169,20 @@ public EntitySite(String title,
         getUserRoles(); // populate the user roles
     }
 
+    public EntitySite(Site site, boolean includeGroups, boolean justInfo) {
```

Comment:
Fair enough. Maybe converting it to a builder pattern would make more sense. Perhaps adding a new method called copyInfoOnly and just using the default constructor?